### PR TITLE
replace validate_caller -> get_sender

### PR DESCRIPTION
**Pull request by dmitryagapov**

### CODE REVIEW

File: openfl/component/director/director.py [hunk 1]
```diff
@@ -88,13 +88,13 @@ async def set_new_experiment(
 
         return True
 
-    def get_trained_model(self, experiment_name: str, caller: str, model_type: str):
+    def get_trained_model(self, experiment_name: str, sender: str, model_type: str):
```
Comment (igor-davidyuk) [inline openfl/component/director/director.py:91]: In my opinion, it is a confusing renaming. `get_trained_model` is a call, so we passed the caller's name as a parameter.
Now we pass the sender's name, and it doesn't make sense, this method will send the model back to the caller, so the sender is the Director in this case

File: openfl/protocols/director.proto [hunk 2]
```diff
@@ -7,142 +7,132 @@ import "google/protobuf/duration.proto";
...
 
 message NodeInfo {
-    string name = 1;
-    string adress = 2;
-    bool cuda_available = 3;
-    uint32 memory_size = 4;
+  string name = 1;
+  string adress = 2;
+  bool cuda_available = 3;
+  uint32 memory_size = 4;
```
Comment (igor-davidyuk) [inline openfl/protocols/director.proto:18]: Doesn't it reduce interface readability?

File: openfl/transport/grpc/director_server.py [hunk 3]
```diff
@@ -54,26 +54,21 @@ def _fill_certs(self, root_ca, key, cert):
...
 
         Args:
-            request: The gRPC message request
             context: The gRPC context
 
         Returns:
-            True - if the caller has valid cert
-            False - if the callers cert name is invalid
+            If TLS is used return sender name from certificate
+            If TLS is unused return default name 'unauthorized_sender'
```
Comment (igor-davidyuk) [inline openfl/transport/grpc/director_server.py:66]: This should be discussed at a separate architectural meeting.

**Review by igor-davidyuk — COMMENTED** [review]
I vote for returning the `caller` back.
Also `unauthorized_sender` part is misleading. Any user that can connect to a Director is authorized to access their userspace. They may be not authenticated without certificates, though

File: openfl/protocols/director.proto (hunk 2 above)
Comment (dmitryagapov) [reply openfl/protocols/director.proto:18]: https://developers.google.com/protocol-buffers/docs/style?hl=ru

**Comment by itrushkin** [comment]
Please merge the latest changes from develop branch in order to pass new CI tests.

**Comment by dmitryagapov** [comment]
will be fixed in https://github.com/intel/openfl/pull/166

**Pull request closed without merging**